content/news/release-20150614.md

```diff
@@ -0,0 +1,51 @@
+We are proud to announce the June 2015 release of OpenRA!
+
+As promised, it was our goal to reduce the length of this development cycle, and we succeeded! With about 22500 changes lines in just over 300 commits by a total of 25 developers since last release, the list of new features is naturally a bit shorter than last time around, but it contains some good ones!
+
+The standout feature of this release is of course the long-awaited in-game map editor. For the first time ever, we are offering a map editing ability for all our mods, on all supported operating systems. You can access the map editor from the "Extras" sub-menu of the main menu.
+
+The new map editor offers the following benefits and features:
+
+* Fully cross-platform, just like the game itself, while the legacy editor did not work on OS X.
```

Comment:
The comment about OS X doesn't really work here.  Maybe move it to parens after the "on all supported operating systems"?